### PR TITLE
[4.0] com_users - remove unused addsubmenu

### PR DESCRIPTION
**Pull request by brianteeman**
Remove unused references to addSubmenu in various views on com_users

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a810d8b01022fcaa86632393c8bc5aeb8b9380aa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25297">issues.joomla.org/tracker/joomla-cms/25297</a>.</sub>

File: administrator/components/com_users/View/Groups/HtmlView.php [hunk 1]
```diff
@@ -81,8 +80,6 @@ public function display($tpl = null)
 		$this->filterForm    = $this->get('FilterForm');
 		$this->activeFilters = $this->get('ActiveFilters');
 
-		UsersHelper::addSubmenu('groups');
```
Comment (wilsonge) [inline administrator/components/com_users/View/Groups/HtmlView.php:84]: You should remove this function too per the other PRs

Comment (Quy) [reply administrator/components/com_users/View/Groups/HtmlView.php:84]: It is not there unless I am looking in the wrong file.

**Comment by wilsonge** [comment]
Thanks

File: administrator/components/com_users/View/Groups/HtmlView.php (hunk 1 above)
Comment (wilsonge) [reply administrator/components/com_users/View/Groups/HtmlView.php:84]: Ahh I removed it 1.5 years ago :D

Comment (brianteeman) [reply administrator/components/com_users/View/Groups/HtmlView.php:84]: lol

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**